src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,10 @@
+%flexik {
```

Comment:
You can find a better name for placeholder, to develop better in the future, ex: flex-center

Reply:
upvoting